test/common/testlib.py

```diff
@@ -1596,6 +1596,7 @@ def terminate_sessions():
 
             # Restart logind to mop up empty "closing" sessions
             self.machine.execute("systemctl restart systemd-logind")
+            self.machine.execute("systemctl restart systemd-user-sessions.service")
```

Comment:
No it does not kill sessions, I just thought it might clean up leftover processes.
```
[root@fedora-37-127-0-0-2-2201 ~]# loginctl --no-legend list-sessions
3    0 root    
4 1000 admin   web console
6    0 root    pts/0
[root@fedora-37-127-0-0-2-2201 ~]# systemctl restart systemd-user-sessions.service
[root@fedora-37-127-0-0-2-2201 ~]# loginctl --no-legend list-sessions
3    0 root    
4 1000 admin   web console
6    0 root    pts/0
```